projects/ad2s1210_iio/tests/func/test_ad2s1210_example.py

```diff
@@ -0,0 +1,70 @@
+import decimal
+import pytest
+import serial
+from time import sleep
+import csv
+import os
+from numpy import ndarray
+from ad2s1210 import *    # TODO - Delete later
+
+iio_device = { 'DEV_AD2S1210': 'ad2s1210',}
+#max angle is 2pi radians
+MAX_EXPECTED_ANGLE = 65535
+MIN_EXPECTED_ANGLE = 0
```

Comment:
Are we validating the readback value from 0 to full scale? The test result would pass for any value between 0-65535 here.

Generally there's a fixed source of input and the validation happens for a specific value that we already know.
Has there been any discussion on what is the source of input here?